lib/src/base-styles/_nx-tables.scss

```diff
@@ -183,6 +183,13 @@
   -ms-user-select: none;
 }
 
+.nx-cell--header--filter {
+  background-color: white;
```

Comment:
Stick to the hex color codes.  At some point we are going to create scss variables again for all the colors we use and at that point we plan to search/replace all the hex literals with the corresponding variable names.

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/4f2e827522b2c9ccfec400e47e4f3529cb43259d